alibaba-dragonwell/get-help.md

```diff
@@ -0,0 +1 @@
+[Dragonwell Containers](https://github.com/dragonwell-containers/dragonwell-containers.git)
```

Comment:
You probably want this to link somewhere like https://github.com/alibaba/dragonwell8/issues instead, right?
(That's where the website appears to suggest for users looking for "support" or "community")

Reply:
Yes. updated to https://github.com/alibaba/dragonwell8/issues